site/blog/authors.yml

```diff
@@ -52,3 +52,12 @@ mathetake:
   socials:
     github: https://github.com/mathetake
     linkedin: https://www.linkedin.com/in/mathetake/
+xunzhuo:
+  name: Xunzhuo (Bit) Liu
+  title: Envoy AI Gateway Maintainer - Tencent
```

Comment:
so Alexa has `Senior Software Engineer, Bloomberg` vs here using "-". Can we stick to either one of them as a separator